R-package/tests/testthat/test_parameters.R

```diff
@@ -91,7 +91,7 @@ test_that(".PARAMETER_ALIASES() uses the internal session cache", {
   expect_false(exists(cache_key, where = .lgb_session_cache_env))
 })
 
-test_that("training should warn if you use 'dart' boosting, specified with 'boosting' or aliases", {
+test_that("training should only warn if you use 'dart' boosting, specified with 'boosting' or aliases", {
```

Comment:
```suggestion
test_that("training should warn if you use 'dart' boosting and requesting early stopping, specified with 'boosting' or aliases", {
```

I think this would be a more accurate description of what this tests.